### PR TITLE
Added basic calculations

### PR DESCRIPTION
**Pull request by sritanmay001**

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Hello there @sritanmay001, good work. I see that you are trying to learn git and GitHub. Always glad to see new contributors on the repository. Will you like to handle some merge conflicts that I can create for you to learn more about them. Or maybe branching? Let me know, I won't merge this without it.

**Comment by sritanmay001** [comment]
@vipulgupta2048 I would love to handle merge conflicts and branching.

**Comment by vipulgupta2048** [comment]
I also expect an explanation on your part, of your thought process and step taken. This would help me, help you to fix your process. Have fun, bruv

File: Hello Programs/hello.py [hunk 1]
```diff
@@ -1 +1,5 @@
-print ("Chocolate")
+<<<<<<< HEAD
+print ("Hello")
+=======
+print ("Hello")
+>>>>>>> fabc02c1827f0a4d53d524e9874639f601e86a9f
```
Comment (vipulgupta2048) [inline Hello Programs/hello.py]: So I see you solve the conflict but now you see these weird lines of gibberish coming in that wasn't there before. If you check out the `Files Changed` tab on the top, and navigate to the changes of this file. You will see earlier the file was `print ("Chocolate")` and was just 1 line. You have 5 lines now. Why is this happening you may think.

It's safe to safe it isn't gibberish and it really means something. This is git's way of telling you that there are basically 2 versions of the file. And you have to choose which version you want to keep. I would really ask you to read through this tutorial given below line by line especially point 5 and 6. 

https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line

If you still don't get it, we will talk more about this. Have fun

Comment (vipulgupta2048) [reply Hello Programs/hello.py]: @sritanmay001 How's it going? Got it resolved yet.

**Comment by sritanmay001** [comment]
@vipulgupta2048 Resolved conflict.

**Comment by sritanmay001** [comment]
@vipulgupta2048 Through the contents of the link and through what you told, I came to understand that there are two versions of this program. So, I edited the program and kept the thing that I thought was needed in the program.

**Pull request merged**